### ISSUE TITLE
Oppositive way

### ISSUE THREAD

**Issue by LuanTeles** [chidreams/Artemis-Patch-Collection-RPCS3#2]
Can you also do the oppositive? i mean the RPCS3 patches for use with the Artemis patch engine on the real hardware.

**Comment by chidreams** [comment]
Yes, you can indeed reverse the process and convert RPCS3 patches for use with the Artemis patch engine on real hardware. The steps involve extracting the necessary codes from RPCS3 and formatting them to be compatible with Artemis.

1. Extract Codes: Identify the patches and codes used in RPCS3.

2. Convert to Artemis Format: Modify the codes to match the Artemis format, often involving converting address formats and ensuring they align with the PS3's memory layout. 

3. Test on Real Hardware: Use Artemis to apply these patches on the actual PS3 hardware to verify they work as intended.

Example: 
PPU-b84046fb82cd48bd2d093c06bb73d3e031b20199:
  "(Career Mode) Cash Increases With Purchase ":
    Games:
      "EA SPORTS™ Fight Night Round 3 (Artemis)":
        BLUS30019: [ 01.00 ]
    Author: ChiDreamS
    Notes: 
    Patch Version: 1.2
    Patch:
    - [ be32,0x00243904,0x916A0170] 
 
So the converted code for Artemis 
(Career Mode) Cash Increases With Purchase
0 00243904 916A0170
 #
The only thing you need to be aware of is the code type for Artemis. From my findings, RPCS3 only supports few types from the official product.
